examples/datasource-http/src/DataSource.ts

```diff
@@ -41,11 +41,20 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
        * }
        */
       const datapoints = response.data.datapoints;
+      if (typeof datapoints === 'undefined') {
```

Comment:
use 

```
if (datapoints === undefined) {
```

instead.

small explanation: `typeof [var] === 'undefined'` is a bit less performant and it is only useful when you are not sure if the variable will be initialized or not. In this case you are sure the variable is initialized (in the previous line) and you can use the small optimization of comparing directly to undefined vs getting the type and comparing strings.